src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        return Color.values()[random.nextInt(Color.values().length)].name();
```

Comment:
It would be easier to read it if you split it into two lines:
```suggestion
        int index = random.nextInt(Color.values().length);
        return Color.values()[index].name();
```